version/info.go

```diff
@@ -32,3 +34,16 @@ var Map = map[string]string{
 	"buildDate": BuildDate,
 	"goVersion": GoVersion,
 }
+
+func init() {
+	buildInfo := prometheus.NewGaugeVec(
```

Comment:
@beorn7 Actually it'd be cool to be able to use a const metric for this. But you can't register that with Prometheus because `ConstMetric()` returns a `Metric`, which doesn't implement `Collector`. Wondering if `constMetric` should be an exported type which also implements the `Collector` interface?